your/utils/rfi.py

```diff
@@ -131,6 +133,8 @@ def sk_sg_filter(
     Args:
         data (numpy.ndarray): 2D frequency time data
         your_object: Your object
+        foff (float): Channel bandwidth (MHz)
```

Comment:
it would be good to mention in the documentation that if both `your_object` and `foff` and/or `tsamp` is provided, the code will use the user-provided `foff` and/or `tsamp` values.

Reply:
https://github.com/thepetabyteproject/your/pull/83/files#diff-95754a8efb570ba353c90b74c1acf3b080c9ffa51e9c6a0f44e9825c8e310002R139